core/src/main/java/org/apache/accumulo/fate/zookeeper/ZooLock.java

```diff
@@ -222,9 +222,9 @@ public int compare(String o1, String o2) {
         }
       });
     }
-    if (LOG.isDebugEnabled()) {
-      LOG.debug("Children nodes: {}", validChildren.size());
-      validChildren.forEach(c -> LOG.debug("- {}", c));
+    if (LOG.isTraceEnabled()) {
+      LOG.trace("Children nodes: {}", validChildren.size());
+      validChildren.forEach(c -> LOG.trace("- {}", c));
     }
```

Comment:
This could also be simplified:
```suggestion
    LOG.trace("Children nodes (size: {}): {}", validChildren.size(), validChildren);
```